src/mappings/index.ts

```diff
@@ -1,5 +1,6 @@
 export { onTokenListing } from './tokens'
 export { onOrderPlacement, onOrderCancellation, onOrderDeletion } from './orders'
-export { onTrade, onTradeReversion } from './trades'
+export { onTrade } from './trades'
+export { onTradeReversion } from './trade_reversions'
```

Comment:
I like that they are split into their own modules now.

Reply:
Yep, good call

nit: I don't think It's very common snake case for TS files. Normally camel case